lib/statics/my_colors.dart

```diff
@@ -13,23 +12,26 @@ class MyColors{
   static const Cyan = 0xFF152b1b;
   static const Green = 0xFFC4EB89;
   static const Blue_Gr = 0xFF5EC4D6;
+  static const ToastGreen = 0xFF00D16C;
```

Comment:
Hier am besten noch ein kurzer Dokumentationskommentar drüber (mit /// vorne und auf **englisch**), damit andere Entwickler schneller verstehen, worum es sich handelt. 

- [ ] Einfach kurzer Satz, dass das die Farbe für die Infonachrichten/toasts ist, die über erfolgreiche Transactions etc. 

[Mehr Infos](https://dart.dev/guides/language/effective-dart/documentation#do-use--doc-comments-to-document-members-and-types)